plugins/pulp_rpm/plugins/distributors/yum/metadata/updateinfo.py

```diff
@@ -72,14 +72,7 @@ def _get_repo_unit_nevra(self, erratum_unit):
                                          version=pkg['version'], release=pkg['release'],
                                          arch=pkg['arch'])
                 if pkg_nevra in self.nevra_in_repo:
-                    pkg_dict = {
-                        'name': pkg_nevra.name,
-                        'epoch': pkg_nevra.epoch,
-                        'version': pkg_nevra.version,
-                        'release': pkg_nevra.release,
-                        'arch': pkg_nevra.arch,
-                    }
-                    nevra_in_repo_and_pkglist.append(pkg_dict)
+                    nevra_in_repo_and_pkglist.append(pkg_nevra._asdict())
```

Comment:
This change admittedly has nothing to do with unique collection names. In the course of my tinkering with this issue, it stood out as a relatively simple tweak worth including.

With regard to potentially breaking el6 compatibility for such a minor change:
```
[pulp@el6 ~]$ python
Python 2.6.6 (r266:84292, May 22 2015, 08:34:51) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-15)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from collections import namedtuple
>>> nt = namedtuple('nt', ['foo', 'bar'])
>>> nt(0, 1)._asdict()
{'foo': 0, 'bar': 1}
```